@here/harp-mapview/lib/Tile.ts

```diff
@@ -262,6 +262,14 @@ export class Tile implements CachedResource {
      */
     skipRendering = false;
 
+    /**
+     * If the tile should not yet be rendered, this is used typically when the tile in question
+     * does not fit into the gpu upload limit of the current frame.
+     * Setting this value directly affects the [[willRender]] method, unless
+     * overriden by deriving classes.
+     */
+    delayRendering = false;
```

Comment:
I wonder if we really need a new flag. Couldn't we reuse `skipRendering`?

Reply:
no, I tried :), this conflicts with the fallback handling, as the skipRendering is used for the fallbacks, and we then cannot differentiate and the gpu upload logig would occasionally override the skipRendering flag, that was set by the fallback handling and vice versa, then i.e. background tiles could be not skipped anymore because there is space to upload them etc.

Reply:
I see